### DISCUSSION TITLE
Windows 10 Start menu not working properly

### DISCUSSION THREAD

**Discussion by SpiritTheWalf** [valinet/ExplorerPatcher#3142] [Q&A] [answered]
![image](https://github.com/valinet/ExplorerPatcher/assets/161088035/e53607c2-388c-4ce0-ac67-589c02ce50de)

I have the Windows 10 start menu enabled, and I still have the Windows 11 one, yes I have restarted explorer, and even my pc multiple times

**Comment by SpiritTheWalf** [comment]
ExplorerPatcher version 22621.3296.64.1
Windows 11 version 23H2 build 22631.3520

**Comment by pyrates999** [comment, marked as answer]
Please install the latest pre-release version of EP since you are running a preview build: https://blogs.windows.com/windows-insider/2024/04/11/releasing-windows-11-builds-22621-3520-and-22631-3520-to-the-release-preview-channel/

**Reply by johnkarlhill** [reply]
I had the same issue and updating to the pre-release worked for me.
